teams/delius-iaps/components/files/login.cmd

```diff
@@ -0,0 +1,2 @@
+Powershell -Command "Set-ExecutionPolicy Unrestricted CurrentUser" >> "%TEMP%\StartupLog.txt" 2>&1
```

Comment:
does this permanently set execution policy to "Unrestricted" for each user who runs this?